packages/@okta/i18n/dist/properties/login.properties

```diff
@@ -296,8 +296,8 @@ enroll.choices.title = Set up multifactor authentication
 enroll.choices.description = Your company requires multifactor authentication to add an additional layer of security when signing in to your Okta account
 enroll.choices.description.generic = Your company requires multifactor authentication to add an additional layer of security when signing in to your account
 enroll.choices.description.specific = Your company requires multifactor authentication to add an additional layer of security when signing in to your {0} account
-enroll.choices.description.gracePeriod = Your company recommends setting up additional factors for authentication. Set up will be required in: {0} day(s).
-enroll.choices.description.gracePeriod.oneDay = Your company recommends setting up additional factors for authentication. Set up will be required in: less than 1 day.
+enroll.choices.description.gracePeriod.bold = Your company recommends setting up additional factors for authentication. Set up will be required in: <b>{0} day(s)</b>.
+enroll.choices.description.gracePeriod.oneDay.bold = Your company recommends setting up additional factors for authentication. Set up will be required in: <b>less than a day</b>.
```

Comment:
you might even be able to just edit the existing ones... Let's check with @santhoshbalakrishnan

Reply:
He replied saying it should be add and move existing strings to deleted section